app/components/media/tag_component.rb

```diff
@@ -87,6 +86,18 @@ def media_tag # rubocop:disable Metrics/MethodLength
       end
     end
 
+    def action
+      action = 'media-seek@window->media-player#seek ' \
+               'auth-success@window->media-player#initializeVideoJSPlayer'
+
+      action += ' fullscreenchange@window->media-player#fullscreenChange' if safari?
```

Comment:
Could we just do this all the time, and put all the conditionals on the front end? Then it's more or less all in the same place rather than spread across front and back ends.

Reply:
I thought about that, but I decided against it because everytime a user goes fullscreen, then this function is going to be run, no matter the browser and that just adds processing that might not be necessary